src/api/app/controllers/webui/request_controller.rb

```diff
@@ -76,13 +76,13 @@ def modify_review
   def show
     diff_limit = params[:full_diff] ? 0 : nil
     @req = @bs_request.webui_infos(filelimit: diff_limit, tarlimit: diff_limit, diff_to_superseded: @diff_to_superseded)
-    @is_author = @bs_request.creator == User.current.login
+    @is_author = @bs_request.creator == User.possibly_nobody.login
     @is_target_maintainer = @req['is_target_maintainer']
 
     @my_open_reviews = @req['my_open_reviews']
     @other_open_reviews = @req['other_open_reviews']
-    @can_add_reviews = @bs_request.state.in?([:new, :review]) && (@is_author || @is_target_maintainer || @my_open_reviews.present?) && !User.current.is_nobody?
-    @can_handle_request = @bs_request.state.in?([:new, :review, :declined]) && (@is_target_maintainer || @is_author) && !User.current.is_nobody?
+    @can_add_reviews = @bs_request.state.in?([:new, :review]) && (@is_author || @is_target_maintainer || @my_open_reviews.present?)
```

Comment:
in case you wonder: I removed the is_nobody? here as `@is_target_maintainer  || @is_author` kind of give it away it's not nobody